Adds robust granular access

IamVelcroboy opened the pull request:
## About the PR
- Adds individual access and airlocks for Boxer, Clown, Library, Mime, Musician, Reporter, Psychologist, and Zookeeper 
- Adds Boxer and Reporter the the Theatre group of accesses
- Adds a Sec/lawyer maintenance hatch
- Adds a salvage locked external airlock
- Updates Rep Drobe and Boxing Drobe with appropriate access
- Removes redundant service from Janitor(was merged upstream)

## Why / Balance
Roles with specific spaces mapped for them should have reasonable expectation of privacy/security in those spaces.

## Technical details
n/a

## Media
n/a

## Breaking changes
n/a

**Changelog**
n/a Primarily a mapping change

Commit:
Adds robust granular access

Commit:
Updates drobes with appropriate access

Commit:
Typo/file not saved

IamVelcroboy committed:
Merge branch 'master' into Accesseroo

Colin-Tel commented on Resources/Prototypes/Access/medical.yml:
```diff
@@ -9,7 +9,7 @@
   id: Chemistry
   name: id-card-access-level-chemistry
-  
+
```
consider un-touching this whitespace

IamVelcroboy replied:
There's nothing there. I'm not even sure how to undo that.

Colin-Tel commented on Content.Shared/Access/Components/IdCardConsoleComponent.cs:
```diff
@@ -64,9 +66,15 @@ public WriteToTargetIdMessage(string fullName, string jobTitle, List<string> acc
+        "Psychologist", // DeltaV - Add Psychologist access
+        "Quartermaster",
+        "Reporter", // DeltaV - Add Psychologist access
         "Quartermaster",
```
```suggestion
        "Reporter", // DeltaV - Add Psychologist access
        "Quartermaster",
```
Duplicate access

Colin-Tel replied:
also a little weird the reporter access has that comment

Colin-Tel reviewed (changes requested):
I think this is something worth trying, only a couple things to change.

Commit:
Remove dupe/copy-paste oops

Commit:
Remove dupe/copy-paste oops

Commit:
Remove dupe/copy-paste oops

Commit:
Revert "Typo/file not saved"

This reverts commit 190363ead8db415e1d0e06e27b884cbcf835a398.

Commit:
Revert "Revert "Typo/file not saved""

This reverts commit dd116f2b04f48d3dad97dc4e57a56a3ed1234e7d.

Commit:
Revert "Remove dupe/copy-paste oops"

This reverts commit 5660903ac46f920efc73bea2f217c5ad50268c96.

Commit:
Remove dupe, fix copy/paste

Commit:
Merge remote-tracking branch 'origin/Accesseroo' into Accesseroo

IamVelcroboy commented:
Fixed the dupe/type but the white-space I can't figure out how to undo. I even deleted the file and copied over a fresh one. It just doesn't register that I've changed anything. So, gives me nothing to commit.

Commit:
Fix some more copy/paste

Commit:
ugg
